common/network-common/src/test/java/org/apache/spark/network/ssl/ReloadingX509TrustManagerSuite.java

```diff
@@ -161,14 +161,17 @@ public void testReload() throws Exception {
       // At this point we haven't reloaded, just the initial load
       assertEquals(0, tm.reloadCount);
 
+      // Wait so that the file modification time is different
+      Thread.sleep((tm.getReloadInterval() + 200));
```

Comment:
Why do we need this ?

Reply:
thanks, fixing.

also, for future people seeing this: the reason this is needed is that in certain scenarios this can run too fast, so the modification time of the original trust store and the one we create right after this is the same - in which case we do not reload the file, which fails this test.